config/routes.rb

```diff
@@ -4,12 +4,18 @@
   get "/help", to: "static_pages#help"
   get "/about", to: "static_pages#about"
   get "/contact", to: "static_pages#contact"
-  get "/manage_user", to: "admin#manage_user"
+  get "/index", to: "admin#index"
   get "/info", to: "details#info"
   get "/signup", to: "users#new"
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
-  resources :users
+  resources :users, except: :destroy
+  namespace :admin do
+    resources :authors, only: %i(index create edit update destroy)
```

Comment:
If there is more than 3 actions, please use except

Reply:
I can write resources :authors ?? because it use all the actions